pkg/machine/convert.go

```diff
@@ -308,8 +309,6 @@ func GetAPIV2NodeCloudSpec(machineSpec clusterv1alpha1.MachineSpec) (*apiv1.Node
 			PrimaryDiskOSImage:          config.VirtualMachine.Template.PrimaryDisk.OsImage.Value,
 			PrimaryDiskStorageClassName: config.VirtualMachine.Template.PrimaryDisk.StorageClassName.Value,
 			PrimaryDiskSize:             config.VirtualMachine.Template.PrimaryDisk.Size.Value,
-			PodAffinityPreset:           config.Affinity.PodAffinityPreset.Value,
-			PodAntiAffinityPreset:       config.Affinity.PodAntiAffinityPreset.Value,
```

Comment:
what about existing md's? If we remove this then the users wont see these values anymore? Or is this somehow migrated automatically to the TopologySpreadConstraints?

Reply:
The idea is that old machine deployments that have `PodAffinityPresets` should be rotated, however, if a user would not do that then the machine should work as it used to be (with pod affinity in place - since it's already working)

Although for the UI we will drop the pod affinity preset field. @sankalp-r if we merge this right now I understand that there will not be error but empty pod affinity preset fields, right? Did you test this?

Reply:
@mfranczy needs to be tested again from the UI/API perspective.

Reply:
@sankalp-r then please do, thanks.

@lsviben 
>  If we remove this then the users wont see these values anymore?

correct, that's the idea, we are in technology preview phase and we want to force users to move to new API, as well we will explain this in the doc.